src/test/java/co/cask/hydrator/salesforce/plugin/source/batch/util/SalesforceQueryUtilTest.java

```diff
@@ -146,4 +146,33 @@ public void testCreateSObjectQueryWithAllFiltersProvided() {
     Assert.assertEquals("SELECT Id,Name,SomeField FROM sObjectName WHERE LastModifiedDate>LAST_WEEK",
                         sObjectQuery);
   }
+
+  @Test
+  public void testIsQueryUnderLengthLimitTrue() {
+    boolean underLengthLimit = SalesforceQueryUtil.isQueryUnderLengthLimit(
+      "SELECT Id,Name,SomeField FROM sObjectName WHERE LastModifiedDate>LAST_WEEK");
+
+    Assert.assertTrue(underLengthLimit);
+  }
+
+  @Test
+  public void testIsQueryUnderLengthLimitFalse() {
+    boolean underLengthLimit = SalesforceQueryUtil.isQueryUnderLengthLimit(
+      "SELECT Id,Name,SomeField FROM sObjectName "
```

Comment:
Is this actually something we wish to consider over the length limit? Or should we do some sort of whitespace removal?

Reply:
This was hack to emulate wide query but I agree it's not the best example in the unit test, I will rewrite unit test to have long field names to avoid the confusion.
Regarding whitespace removal, we do remove spaces in the beginning and in the end of the query (using `trim()` function), though removing spaces inside the query might be error-prone.

Reply:
I see - I'm ok with it for just the unit test - but do we plan on having an integration test for wide objects in the future?

Reply:
Yes, I'm planning to add integration tests in next PR.